app/app.js

```diff
@@ -3,7 +3,6 @@ const config = require('../config')
 const express = require('express')
 const nunjucksSetup = require('./services/nunjucks-setup')
 const path = require('path')
-const favicon = require('serve-favicon')
```

Comment:
New GDS has favicon in layout already so can remove this - and have taken the module out of `package.json`.